### PR TITLE
NW4/Gulnihal Naldoken/ HTML-CSS-Week 3

### PR DESCRIPTION
**Pull request by gulnihaln**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,10 +7,58 @@
...
+            <img 
+            src="/img/mint-cupcake-logo2.png" 
+            alt="logo"
+            width="100%"
+            height="100%">
+        </div>
+        <div class="nav-bar">
+                <label for="toggle">&#9776;</label>
+                <input type="checkbox" id="toggle"/>
+                <div class="menu">
```
Comment (zeinaji) [inline index.html:27]: You could use a `nav` tag here instead of `div` to make it more semantic

File: index.html [hunk 2]
```diff
@@ -7,10 +7,58 @@
...
 </head>
-
 <body>
     <!-- Add your markup here -->
-</body>
-
+    <header class="header-container">
+        <div class="logo">
+            <img 
+            src="/img/mint-cupcake-logo2.png"
```
Comment (zeinaji) [inline index.html:19]: Having the same problem here of pictures not appearing until I change this to `img/mint-cupcake-logo2.png` 😩 just curious how this is working for you but not for me 🤔

**Review by zeinaji — COMMENTED** [review]
Great work as always, description followed and website looks great 🙂

File: index.html (hunk 1 above)
Comment (gulnihaln) [reply index.html:27]: Thanks Zein, I made it with nav, but I got help here and she helped me like that. it was in my mind, I'll return it to nav.

File: index.html (hunk 2 above)
Comment (gulnihaln) [reply index.html:19]: Ohh Sorry. It stays there when I "/" and call the link. I'll correct them.

**Pull request closed without merging**